website/common/locales/en/groups.json

```diff
@@ -35,6 +35,8 @@
   "leave": "Leave",
   "invitedTo": "Invited to <%= name %>",
   "invitedToNewParty": "You were invited to join a party! Do you want to leave this party and join <%= partyName %>?",
+  "partyInvitationsText": "You have <%= numberInvites %> party invitations! Choose wisely, because you can only be in one party at a time.",
+  "joinPartyConfirmationText": "Are you sure you want to join the party \"<%= partyName %>\"? You can only be in one party at a time. If you join, all other party invitations will be cancelled.",
```

Comment:
Agreed. That makes it clear that the action taken automatically is the same as can be done manually.